chaincfg/params.go

```diff
@@ -386,7 +401,7 @@ var TestNetParams = Params{
 	// Chain parameters
 	GenesisBlock:             &testNetGenesisBlock,
 	GenesisHash:              &testNetGenesisHash,
-	CurrentBlockVersion:      0,
+	CurrentBlockVersion:      1,
```

Comment:
I believe 2 makes more sense.  The main network is moving to version 2, so they should all be consistent imo.